### DISCUSSION TITLE
Protocol Gcs vs Gs

### DISCUSSION THREAD

**Discussion by parthspatel-mendel** [dask/gcsfs#580] [Q&A]
Why do you provide preference to the protocol being gcs instead of gs?  From google's documentation, shouldn't we prefer gs?
https://github.com/fsspec/gcsfs/blob/329f6d877df17859346f0e0ecdf53985badc539c/gcsfs/core.py#L264

**Comment by martindurant** [comment]
I believe "gcs" was first, before google had any feelings on the matter. Also, it's in the name of the package.
